### PR TITLE
RemoveAttribute doesn't work on SVG elements in IE 10.  Use setAttrib…

### LINKED ISSUE #871: Dragging blocks does not work in IE10
Block dragging appears to be offset by the scrollbar delta in IE10, rendering Blockly almost useless in IE10 as as soon as you click on a block, it jumps away from the mouse such that it's off the screen. See screen capture video: https://s3-ap-southeast-2.amazonaws.com/grok-uploads/bug-reports/blockly-IE10-block-dragging-bug.mp4

### PR DESCRIPTION
**Pull request by rachel-fenichel**
…ute to null instead.

Fixes #871!

### CODE REVIEW

File: core/utils.js [hunk 1]
```diff
@@ -38,6 +38,23 @@ goog.require('goog.events.BrowserFeature');
...
+ */
+Blockly.utils.removeAttribute = function(element, attributeName) {
+  // goog.userAgent.isVersion is deprecated, but the replacement is
+  // goog.userAgent.isVersionOrHigher.
+  if (goog.userAgent.IE && goog.userAgent.isVersion('10.0')) {
+    element.setAttribute(attributeName, null);
+  } else {
+    element.removeAttribute(attributeName);
+  }
+};
```
Comment (picklesrus) [inline core/utils.js]: Do you know if the call to getAttribute after calling removeAttribute vs setAttribute null will return the same thing?  If they're different I can easily see us breaking IE10 again by trying to read the attribute later and it returning something different than the other browsers. No ideas how to solve this... but wondering whether I need to at least keep it in mind.

Comment (rachel-fenichel) [inline core/utils.js]: getAttribute returns the empty string after calling setAttribute to null.
...
...
...

**Comment by rachel-fenichel** [comment]
Merged, but I may also want to cherry-pick this into master to unbreak IE10 there.

**Comment by picklesrus** [comment]
Sure! There have only been 2 changes to develop (this one and one in accessible) so we could just do the whole thing if cherrypicking is messy.

**Comment by rachel-fenichel** [comment]
That sounds good too.

**Pull request merged**